tnoodle-ui/src/ManageCompetition.js

```diff
@@ -126,9 +126,11 @@ class ManageCompetition extends Component {
       </div>;
     } else {
       let disabled, title;
-      if(hasMbld && !puzzlesPer333mbfAttempt) {
+      const WCA_MINIMUM_MBLD_PER_ATTEMPT = 2;
+      // perhaps we should set also an upper bound to avoid numbers like 2e3
+      if(hasMbld && (!puzzlesPer333mbfAttempt || puzzlesPer333mbfAttempt < 2)) {
```

Comment:
Can we change this to reuse the variable you just declared?

```suggestion
      if(hasMbld && (!puzzlesPer333mbfAttempt || puzzlesPer333mbfAttempt < WCA_MINIMUM_MBLD_PER_ATTEMPT)) {
```